test/plugin-nested-docs/int.spec.ts

```diff
@@ -110,6 +111,36 @@ describe('@payloadcms/plugin-nested-docs', () => {
       // @ts-ignore
       expect(lastUpdatedChildBreadcrumbs[0].url).toStrictEqual('/11-children-updated')
     })
+
+    it('should return breadcrumbs as an array of objects', async () => {
+      const parentDoc = await payload.create({
+        collection: 'pages',
+        data: {
+          title: 'parent doc',
+          slug: 'parent-doc',
+          _status: 'published',
+        },
+      })
+
+      const childDoc = await payload.create({
```

Comment:
So previously the update was appending the child data as well 

```
data: {
  ...child,
  [breadcrumbSlug]: await populateBreadcrumbs(req, pluginConfig, collection, child),
},
```

In this test we should perform an update as well and then test that other data from the child doc doesn't get updated as part of this.

Reply:
Absolutely I'll add an additional test. For context, the `populateBreadcrumbs` function is already returning the full doc data, so essentially we were doing this twice and double nesting.

<img width="544" alt="Screenshot 2025-02-20 at 12 43 51 PM" src="https://github.com/user-attachments/assets/17cda644-1ed7-4bcf-8ab2-8a6cf042ae28" />